test/e2e/agent/agent_test.go

```diff
@@ -29,30 +29,10 @@ var _ = Describe("VM Agent behavior", func() {
 	BeforeEach(func() {
 		harness = e2e.NewTestHarness()
 		err := harness.VM.RunAndWaitForSSH()
-		if err != nil {
```

Comment:
I have refactored this so it becomes automatically available to any tests using the harness